Formula/cargo-zigbuild.rb

```diff
@@ -25,7 +25,7 @@ def install
   end
 
   test do
-    # https://github.com/ziglang/zig/issues/10377
+    # Remove errant CPATH environment variable
```

Comment:
Changed the comment because of 

```
Full audit cargo-zigbuild --online --git --skip-style output
  cargo-zigbuild:
    * Formula refers to a GitHub issue or pull request that is closed: https://github.com/ziglang/zig/issues/10377
  Error: 1 problem in 1 formula detected
```